checks/basic/latest_tag_test.go

```diff
@@ -154,6 +155,11 @@ func TestLatestTagWarning(t *testing.T) {
 			objs:     initContainer("busybox:v1.2.3"),
 			expected: nil,
 		},
+		{
+			name:     "pod with invalid iamge name",
```

Comment:
Add a similar test for container too? 
And iamge => image

Reply:
Thanks - addressed both of these.